### PR TITLE
Modify call of forward function in docstring example

### PR DESCRIPTION
**Pull request by vcuspinera**

### CODE REVIEW

File: feature_selection/forward_selection.py [hunk 1]
```diff
@@ -36,8 +36,7 @@ def forward_selection(scorer, X, y, min_features=1, max_features=10):
     >>> from sklearn.datasets import make_friedman1
     >>> data, target = make_friedman1(n_samples=200, n_features=15,
     >>> random_state=0)
-    >>> from feature_selection.forward_selection \
-    >>>      import forward_selection
+    >>> from feature_selection import forward_selection
```
Comment (techrah) [inline feature_selection/forward_selection.py:39]: Actually, I think it's correct the way it was. We are investigating this now. It is related to whether the appropriate imports are in the `__init__.py` file but they are currently causing the build to fail.

Comment (vcuspinera) [reply feature_selection/forward_selection.py:39]: Ok, sounds good Ryan.
Just when accepting or rejecting the PR, please take into consideration that these are two changes and not only one:
1) in the name of the example of the function, and
2) erase the backslash (“\”) that was affecting the example in the docs (as you can see in the image  the import `forward_selection` got at the right after the `>>>`)
image

Comment (techrah) [reply feature_selection/forward_selection.py:39]: No worries. Once I merge this PR, that will be fixed.

**Comment by techrah** [comment]
Nice catch!

**Pull request merged**